pyscriptjs/src/plugin.ts

```diff
@@ -57,19 +61,30 @@ export class Plugin {
 
 export class PluginManager {
     _plugins: Plugin[];
+    _pythonPlugins: any[];
 
     constructor() {
         this._plugins = [];
+        this._pythonPlugins = [];
     }
 
     add(...plugins: Plugin[]) {
         for (const p of plugins)
             this._plugins.push(p);
     }
 
+    addPythonPlugin(plugin: any){
+        this._pythonPlugins.push(plugin);
+    }
+
     configure(config: AppConfig) {
         for (const p of this._plugins)
             p.configure(config);
+
+        for (const p of this._pythonPlugins)
+            if (typeof p.configure !== 'undefined') {
+                p.configure(config);
```

Comment:
You can do (in JavaScript, so I think it'll work in TypeScript):

```JS
p.configure?.(config);
```

Which will only be called if the `p.configure` method exists (saves the conditional).